kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorNetworkIT.java

```diff
@@ -60,20 +64,19 @@
   private static final String EXPECTED_FILE_EXTENSION = "json";
   private static final List<String> KAFKA_TOPICS = Arrays.asList(TEST_TOPIC_NAME);
   private static final int FLUSH_SIZE = 200;
-  private static final int EXPECTED_PARTITION = 0 ;
+  private static final int EXPECTED_PARTITION = 0;
   private static SquidProxy squid;
 
   @Before
-  public void setup() {
-    startConnect();
+  public void initializeResources() {
     createS3RootClient();
     // create the test bucket
     createS3Bucket(TEST_BUCKET_NAME);
   }
 
```

Comment:
We still need to keep the `@After` method you had previously, but only with `emptyBucket` to ensure the bucket is cleared after every test. `@AfterClass` is for the final cleanup, so we need both.